fsw/src/sch_lab_app.c

```diff
@@ -128,7 +122,7 @@ void SCH_Lab_AppMain(void)
                     if (LocalStateEntry->Counter >= LocalStateEntry->PacketRate)
                     {
                         LocalStateEntry->Counter = 0;
-                        CFE_SB_SendMsg(&LocalStateEntry->MsgBuf.MsgHdr);
+                        CFE_SB_SendMsg((CFE_SB_Msg_t *)&LocalStateEntry->MsgBuf);
```

Comment:
No need to cast here, just use `&LocalStateEntry->MsgBuf.BaseMsg` (or equivalent).